knimin/lib/data_access.py

```diff
@@ -3068,17 +3072,15 @@ def read_sample_plate(self, id):
         Returns
         -------
         dict
-            {name : str, email : str, created_on: datetime, notes : str,
-             plate_type : str}
-            Properties of the sample plate: name, who created it, when it was
-            created, notes, and plate type name
+            {name : str, plate_type_id : int, email : str,
+             created_on: datetime, notes : str}
```

Comment:
I don't think this comment should be addressed in this PR - but in the future this return type is a bit confusing.

Suggest changing this to NamedTuples instead of dictionaries.  Not not clear if the ordering matters, or what are the types for the keys, or what are the names of the keys ...